README.md

```diff
@@ -11,7 +11,9 @@ The project is built on top of NodeJS framework. All the regular nuances for a N
 | Typescript | 4.8.2 |
 
 ### Local setup
-Step 1: Create .env file in root folder with below variables and specify values as required
+---
+
+Step 1: Application configuration is read from .env file. Below are the list of environemnt variables service is dependent on. An example of environment file is available [here](./env.example) and description of environment variable is presented in below table
```

Comment:
Step 3 ->should be on a separate line.
Step 4 -> should be on a separate line.
Step 5 -> should be on a separate line.